src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -69,6 +70,19 @@ private ConversationStore(PersistentStorageAgent persistentStorageAgent) {
     return conversations;
   }
 
+  /** Access the current set of conversations known to the user. */
+  public List<Conversation> getAvailableConversations(User user) {
+    List<Conversation> availableConversations = new ArrayList<>();
+    for (Conversation conversation : conversations){
+      if (conversation.isPrivate()){
```

Comment:
can you add comments to explain the logic here

if a conversation in conversations is private, that means the user knows about it because they were added to it?  Why is there no check whether user exists or if user is in the conversation? 

I know I'm missing background context so if lmk if I'm misinterpreting the conversations variable.

Reply:
Yeah your point is actually correct. This is a bug that I am going to fix in the next commit.